quadtree/quadtree.go

```diff
@@ -378,8 +416,91 @@ func (v *findVisitor) Visit(p geo.Pointer) {
 		y := v.point.Y()
 		v.closestBound.Set(x-d, x+d, y-d, y+d)
 	}
+}
+
+type pointsQueueItem struct {
+	point    geo.Pointer
+	distance float64 // distance to point and priority inside the queue
+	index    int     // point index in queue
+}
+
+type pointsQueue []pointsQueueItem
+
+func NewPointsQueue(capacity int) pointsQueue {
```

Comment:
This method should not be exported. Push expects a `pointsQueueItem` which isn't exported.

Reply:
Yes, it shouldn't. Fixed.